xmldiff/main.py

```diff
@@ -113,6 +113,13 @@ def make_diff_parser():
     parser.add_argument(
         "--fast-match", action="store_true", help="A faster, less optimal match run."
     )
+    parser.add_argument(
+        "--ignored-attrs",
+        type=str,
+        nargs="?",
+        help="A comma separated list of attributes "
+        "that should be ignored in comparison. Can be empty.",
```

Comment:
It's not a required attribute, so we don't need to say that it can be empty, it's implied.

Reply:
Removed